src/coreclr/nativeaot/Runtime/EHHelpers.cpp

```diff
@@ -201,10 +201,10 @@ struct EXCEPTION_REGISTRATION_RECORD
 };
 #endif // HOST_X86
 
-EXTERN_C void __cdecl RhpFailFastForPInvokeExceptionPreemp(intptr_t PInvokeCallsiteReturnAddr,
-                                                           void* pExceptionRecord, void* pContextRecord);
-EXTERN_C void REDHAWK_CALLCONV RhpFailFastForPInvokeExceptionCoop(intptr_t PInvokeCallsiteReturnAddr,
-                                                                  void* pExceptionRecord, void* pContextRecord);
+PREEMPT_PINVOKE_CDECL_HELPER(void, RhpFailFastForPInvokeExceptionPreemp, (intptr_t PInvokeCallsiteReturnAddr,
```

Comment:
These methods are just like QCALLs in CoreCLR. Can it use the exact same style and convention as QCALL in CoreCLR?

The QCALLs in CoreCLR use the default platform convention. Our strategy to deal with the win x86 calling convention special casing has been to minimize the number of places where the calling convention needs to be explicitly specified and use the default as much as possible. It is why QCALLs in CoreCLR use the default.

Explicit cdecl on these methods is left over from the ProjectN plumbing.

Reply:
Fine. If default (stdcall) is preferred, I can switch to it.